node/consensus/follower.go

```
@@ -176,7 +229,20 @@ func (ce *ConsensusEngine) processBlockProposal(ctx context.Context, blkPropMsg
 
 	if err := ce.validateBlock(blkPropMsg.blk); err != nil {
 		ce.log.Error("Error validating block, sending NACK", "error", err)
-		go ce.ackBroadcaster(false, blkPropMsg.height, blkPropMsg.blkHash, nil, nil)
+		sig, err := types.SignVote(blkPropMsg.blkHash, false, nil, ce.privKey)
+		if err != nil {
+			ce.log.Error("Error signing the voteInfo", "error", err)
```

We saw this ignored error from SignVote elsewhere recently.  Are we sure we want to send it with an invalid sig?

---

Ah no, definitely not. the error return is missing here, thought I fixed this in some PR. Anyways, fixing it here.